desktop-exporter/app/components/detail-view/fields-pannel.tsx

```diff
@@ -29,8 +29,72 @@ export function FieldsPannel(props: FieldsPannelProps) {
       </TabPanel>
     );
   }
+
+  // Root span: label with a little tag
+  let isRoot = span.parentSpanID.length ? false : true;
+  let rootTag = isRoot ? (
+    <Tag
+      marginStart={2}
+      colorScheme="cyan"
+      variant="subtle"
+    >
+      root
+    </Tag>
+  ) : null;
+
+  // parentSpanID: omit field if the span is root
+  let parentSpanIDField = !isRoot ? (
+    <SpanField
+      fieldName="parent span id"
+      fieldValue={span.parentSpanID}
+    />
+  ) : null;
```

Comment:
This might be a little easier to read if you flip the order

```jsx
  let parentSpanIDField = isRoot ? 
    null : (
    <SpanField
      fieldName="parent span id"
      fieldValue={span.parentSpanID}
    />
  );
```